polus-ome-zarr-converter-plugin/src/main.py

```diff
@@ -0,0 +1,92 @@
+from bfio.bfio import BioReader, BioWriter
+from preadator import ProcessManager
+
+import argparse, logging
+import numpy as np
+import typing
+from pathlib import Path
+
+# Initialize the logger
+logging.basicConfig(format='%(asctime)s - %(name)-8s - %(levelname)-8s - %(message)s',
+                    datefmt='%d-%b-%y %H:%M:%S')
+logger = logging.getLogger("main")
+logger.setLevel(logging.INFO)
+
+TILE_SIZE = 2**13
+
+def image_to_zarr(inp_image: Path,
+                  out_dir: Path):
+    
+    with ProcessManager.process():
+    
+        with BioReader(inp_image,backend='java') as br:
+            
+            # Loop through timepoints
+            for t in range(br.T):
+
+                # Loop through channels
+                for c in range(br.C):
+            
+                    extension = ''.join(inp_image.suffixes)
+                    
+                    out_path = out_dir.joinpath(inp_image.name.replace(extension,'.ome.zarr'))
+                    if br.C > 1:
+                        out_path = out_dir.joinpath(inp_image.name.replace(extension,f'_c{c}.ome.zarr'))
+                    if br.T > 1:
+                        out_path = out_dir.joinpath(inp_image.name.replace(extension,f'_t{c}.ome.zarr'))
```

Comment:
possible error : `c` has been used instead of `t` in `replace(extension,f'_t{c}.ome.zarr'`